pkg/minikube/download/image.go

```diff
@@ -168,17 +169,40 @@ func ImageToCache(img string) error {
 	}
 }
 
+func parseImage(img string) (*name.Tag, name.Reference, error) {
+	digest, err := name.NewDigest(img)
+	if err == nil {
+		tag := digest.Tag()
+		return &tag, digest, nil
+	}
+
+	_, ok := err.(*name.ErrBadName)
```

Comment:
why we are only checking for this specific errors ? 
add comment when does this specific error happen ?
seems like we are ignoring the error if it is not ErrBadName ?

Reply:
Exactly. `ErrBadName` means `img` contains no digest. It happens if its value is `name:tag` for example. In this case we want to give it a second chance and try to parse it one more time using `name.NewTag(img)`

Reply:
lets add this as a comment

Reply:
done